google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -432,6 +432,8 @@ def delete force: nil
         #   length is 1,024 characters.
         # @param [String] name A descriptive name for the table.
         # @param [String] description A user-friendly description of the table.
+        # @param [EncryptionConfiguration] encryption_configuration A
```

Comment:
Does this need to be an additional argument on this method? We are trying to move away from adding additional arguments to methods like these and instead make these configurations on the table object yielded from the method.

Reply:
It looks to me like there is a create followed by an update in this method. You can update the key parameter, but only if it was originally set when the table was created.

Reply:
Nevermind. I see that if you provide a block, it calls the updater before inserting. Will update this PR